### ISSUE TITLE
Problems with the new AlphaFold Colab Version

### ISSUE THREAD

**Issue by RosaLuzia** [deepmind/alphafold#663]
I run AlphaFold Colab with a lot of similar peptides which are characterized by alpha helices and beta sheets. I always get nice structures with the right assumed conformation, but since the new version of AlphaFold Colab is released structure prediction went wrong. I attach two predictions of the same sequence, one is predicted with an older version and the other with the recent version. Any suggestion how to fix the problem?

![AlphaFold_new](https://user-images.githubusercontent.com/61228596/208880979-6c9ddb05-6b59-45fd-a85b-f9cdfcd55e63.png)
![AlphaFold_old](https://user-images.githubusercontent.com/61228596/208880995-2a0ae4b4-086a-4e3a-ab83-57ce6a20f6cc.png)

**Comment by joshabramson** [comment]
Is this affecting only one or two of your structures, or is there a widespread degradation in quality?

Monomer models should be unchanged by v.2.3.0, but the msa databases used to generate the inputs to the monomer model have been updated. it is possible that for some targets this can reduce the quality of the msa, but for most targets it should be the same or better. Have the sizes of the msa results reduced for the structures where you are having problems?

For this particular target, we found that running with the full local installation, rather than in the colab, gave better results. The full installation uses templates and a larger bfd database, both of which can improve quality.
